### ISSUE TITLE
Cloning options does not copy over any attributes

### ISSUE THREAD

**Issue by zenweasel** [reactioncommerce/reaction#4021]
## Issue Description
When you "clone" an option it should copy over all the values but it doesn't copy over any, so essentially it does nothing that clicking "new" doesn't do.

## Steps to Reproduce
1. Create a product with a variant and an option
1. Click on the "clone" icon
1. Observe that all values are not the same as the original

### Versions
```
Node: 8.9.4
NPM: 5.6.0
Meteor Node: 8.9.4
Meteor NPM: 5.6.0
Reaction CLI: 0.29.0
Reaction: 1.10.0
Reaction branch: release-1.10.0
Docker: 17.12.0-ce
```

**Comment by prinzdezibel** [comment]
@zenweasel  This happens only when cloning a new product with unpublished changes. This is because cloning doesn't clone the revisions, but the latest published version (which is empty for newly created options). I wouldn't consider this behaviour a bug. Going to close. Feel free to reopen if you feel there's more to say or think this is not appropriate.

**Comment by zenweasel** [comment]
When other than when you are creating a product would this feature even have relevance? In addition, this used to work when creating new products.

**Comment by prinzdezibel** [comment]
After some investigation I would say that this feature has never worked as intended. I couldn't find the code that does clone unpublished revisions, even Brent has a vivid memory of seeing this working in the past. I did checkout v1.2 to test it again with an older version, but what I've found is that options (standalone) couldn't be cloned at all (no clone button). Variants do have a clone button, but unpublished changes aren't cloned either. 

On the other hand I somewhat hesitate to introduce new code if there's a chance that the relevant functionality is buried somewhere in the current code state. Talked to Brent and we agreed to icebox this for the time being (until we know more).
